src/main/kotlin/org/rust/lang/core/psi/impl/RustNamedElementImpl.kt

```diff
@@ -6,12 +6,14 @@ import com.intellij.psi.PsiNameIdentifierOwner
 import org.rust.lang.core.psi.RustNamedElement
 import org.rust.lang.core.psi.RustPsiFactory
 import org.rust.lang.core.psi.RustTokenElementTypes
+import org.rust.lang.core.psi.RustTokenType
 
-abstract class RustNamedElementImpl(node: ASTNode) : RustCompositeElementImpl(node),
-                                                     RustNamedElement,
-                                                     PsiNameIdentifierOwner {
+abstract class RustNamedElementImpl(
+    node: ASTNode,
+    val idTokenType: RustTokenType = RustTokenElementTypes.IDENTIFIER
+) : RustCompositeElementImpl(node), RustNamedElement, PsiNameIdentifierOwner {
 
-    override fun getNameIdentifier(): PsiElement? = findChildByType(RustTokenElementTypes.IDENTIFIER)
```

Comment:
As an aside, ideally such "one-mehod" overrides should work via "method injection": https://github.com/JetBrains/Grammar-Kit/blob/master/HOWTO.md#34-implement-interface-via-method-injection, but I have not yet figured out how to make this work with our build system.  "no need to specify arguments or return type" means that grammar kit uses IDEA API to fetch the signatures, which does not exactly work in standalone mode...